lib/main_common.pm

```diff
@@ -501,12 +501,13 @@ sub load_extra_tests() {
             # sysauth test scenarios run in the console
             loadtest "sysauth/sssd";
         }
-        # finished console test and back to desktop
-        loadtest "console/consoletest_finish";
         # kdump is not supported on aarch64, see BSC#990418
         if (!check_var('ARCH', 'aarch64')) {
```

Comment:
Hi, consoletest_finish is moved to the end. Because often it couldn't finish, and then the following tests will be skipped. Thanks for the hint about bug reference, I'll have a look.